WeArtMessageSerializer.cpp

```diff
@@ -1,6 +1,6 @@
 #include "WeArtMessageSerializer.h"
 #include <sstream>
-#include "nlohmann/json.hpp"
+#include <nlohmann/json.hpp>
```

Comment:
This will break compilation in Visual Studio I guess, there is any reason for changing this?

Reply:
I have restored it as it was before